dbms/src/Flash/Mpp/ExchangeReceiver.cpp

```diff
@@ -453,6 +509,8 @@ void ExchangeReceiverBase<RPCContext>::reactor(const std::vector<Request> & asyn
 {
     using AsyncHandler = AsyncRequestHandler<RPCContext, enable_fine_grained_shuffle>;
 
+    addAsyncConnectionNum();
```

Comment:
only add `live_connections` by one here?

Reply:
> only add `live_connections` by one here?

I will fix this